src/sentry/tasks/summaries/utils.py

```diff
@@ -117,15 +148,20 @@ def project_key_errors(ctx: OrganizationReportContext, project) -> list[dict[str
             orderby=[OrderBy(Function("count", []), Direction.DESC)],
             limit=Limit(3),
         )
-        request = Request(dataset=Dataset.Events.value, app_id="reports", query=query)
```

Comment:
The only change here is passing in the referrer so it can be different between the weekly report and the daily summary (as requested by SNS). In doing so I needed to pass the `tenant_ids` param to the request.